packages/neo-one-website/src/components/reference/utils.tsx

```diff
@@ -0,0 +1,35 @@
+// tslint:disable
+import '../../../static/css/prism.css';
+// @ts-ignore
+import Prism from 'prismjs/components/prism-core';
```

Comment:
Change this to import from a common module. Move all of the `Prism` related stuff (basically everything between the tslint:disable and tslint:enable) from https://github.com/neo-one-suite/neo-one/blob/master/packages/neo-one-website/src/elements/Markdown.tsx#L12 to a separate file (in `utils` I suppose) and then have it export `Prism`. Then everywhere else, i.e. in `Markdown.tsx` and here, import `Prism` from the file.

This ensures that `Prism` is setup correctly before it's used by any components.